dev-build/conf/routes

```diff
@@ -120,6 +120,9 @@ GET            /video/end-slate/series/*seriesId.json
 GET            /video/end-slate/section/*sectionId.json                                                                          controllers.VideoEndSlateController.renderSection(sectionId)
 GET            /embed/card/*path.json                                                                                            controllers.RichLinkController.render(path)
 GET            /embed/card/*path                                                                                                 controllers.RichLinkController.renderHtml(path)
+GET            /embed/contentcard/*path.json                                                                                     controllers.ContentCardController.render(path, group: Option[Int], card: Option[Int])
+GET            /embed/contentcard/*path                                                                                          controllers.ContentCardController.renderHtml(path)
```

Comment:
I thought initially that only the render json route was required so I thought I'd have to go back to render html later if required. I found it useful so I should make sure it can accept group and card params too. Good point. 

I already tried briefly to replace the matcher *path with an expression that matches an id without swallowing the query parameters. I forgot to go back and figure it out!